gradlew

```diff
@@ -83,7 +83,7 @@ esac
 CLASSPATH=$APP_HOME/gradle/wrapper/gradle-wrapper.jar
 
 
-# Determine the Java command to use to start the JVM.
+# Determine the Java donjinkrawler.command to use to start the JVM.
```

Comment:
I presume this was changed when you moved the command package to the donjinkrawler package.

Reply:
:(